### ISSUE TITLE
Setting: Control where results are displayed

### ISSUE THREAD

**Issue by mikeburgh** [dbcodeio/public#77]
Add a setting that lets users select where results from user based queries are displayed.. Currently they always go to the panel. 

The setting should provide an option for either the pannel or split vertical or horizontal of the current editor group. 

When the setting is live, and not set, prompt users who execute to confirm where they want results to go by default, and explain that a keyboard modifier (alt or shift pending on mappings) will override the setting and use the alternative option.

System executions (DDL for example) should always still go to the panel.

**Comment by RobertOstermann** [comment]
This would be super nice to have. I always prefer having query results in a separate editor instead of the panel, so having this as an option would be great!

**Comment by mikeburgh** [comment]
This is now available in 1.10.5. When you first execute a query it will now prompt for the result location, and you can change it anytime in settings.

**Comment by RobertOstermann** [comment]
@mikeburgh Seems like having the setup `"dbcode.resultLocation": "beside",` still has the bottom panel pop up, but then puts the results in the side editor as expected.

**Comment by mikeburgh** [comment]
Ugh, indeed.. good catch.. I didn't notice it as I always have that area up, and usually focused on the panel. 

Just pushed a fix, new version should appear shortly in the marketplace.

**Comment by RobertOstermann** [comment]
Thanks for the quick fix!

**Comment by mikeburgh** [comment]
Should be there now

**Comment by RobertOstermann** [comment]
That worked!
